guides/v2.2/cloud/trouble/robots-sitemap.md

```diff
@@ -60,3 +60,45 @@ Create a custom VCL snippet to rewrite the path for `sitemap.xml` to `/media/sit
   "content": "if ( req.url.path ~ \"^/?sitemap.xml$\" ) { set req.url = \"/media/sitemap.xml\"; }"
 }
 ```
+
+See the example below to understand how to rewrite the path for `robots.txt` and `sitemap.xml` to `/media/robots.txt` and `/media/sitemap.xml` accordingly in the single snippet:
+
+```json
+{
+  "name": "sitemaprobots_rewrite",
+  "dynamic": "0",
+  "type": "recv",
+  "priority": "90",
+  "content": "if ( req.url.path ~ \"^/?sitemap.xml$\" ) { set req.url = \"/media/sitemap.xml\"; } else if (req.url.path ~ \"^/?robots.txt$\") { set req.url = \"/media/robots.txt\";}"
+}
+```
+
+#### To use a Fastly VCL snippet for particular domain redirect:
+
+Create a `pub/media/domain_robots.txt` file, where the domain is `domain.com` and use the next VCL snippet:
+
+```
+{
+  "name": "domain_robots",
+  "dynamic": "0",
+  "type": "recv",
+  "priority": "90",
+  "content": "if ( req.url.path == \"/robots.txt\" ) { if ( req.http.host ~ \"(domain).com$\" ) { set req.url = \"/media/\" re.group.1 \"_robots.txt\"; }}"
+}
+```
+
+VCL will take care of routing `http://domain.com/robots.txt` and present the correct file.
```

Comment:
remove "will"

"takes care of" is very subjective. It is possible you could say " The VCL snippet routes and presents the ...file."

Reply:
**done**